va_explorer/users/models.py

```diff
@@ -58,6 +59,21 @@ def __init__(self, *args, **kwargs):
 
     location_restrictions = ManyToManyField(Location, related_name="users", db_table="users_user_location_restrictions")
 
```

Comment:
@pkmitre At standup earlier this week, I asked Matt and Brendan what was meant by Choolwe's description here: "Varied view of system users (modifiable only by Admins)." This bullet is part of one of the tickets I was assigned and comes from Choolwe's Word document. Matt and Brendan mentioned that this refers to geographic scoping. Based on that, it seems like we also need geographic scoping on the Users page.

Reply:
That makes sense; only admins should see the user management page, but there may be value in allowing "localized" admins. They would see only users for their locations, and when creating a user be limited in what locations they can be assigned to. While I think this is something we want to implement, I suggest that it not be in the highest priority tier of work.

Reply:
Created [#177124888](https://www.pivotaltracker.com/story/show/177124888) to handle this work in the future.